roles/edpm_nova/tasks/configure.yml

```diff
@@ -12,65 +13,65 @@
     group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   with_items:
```

Comment:
Since this PR is about bringing the role up to the standard of the repo, it might be nice to use loop instead of with_items.

Reply:
Thanks @bshephar !

Reply:
why woud we prefer loop over with items?

Reply:
ya i just found that.
i general dislike the loop syntax but i can try an d use it.
form an understanding point of view not haveing typed loops i feel is more error prone but
if that is the new way i guess i can try an unlern my old ansibel preferences for using with_*

Reply:
> why woud we prefer loop over with items?

Just for coherence with the rest of the repo. We started using `loop` while importing roles so we're using as a standard (where possible, obviously).  BTW, just to be fair, it isn't deprecated (as stated in the notes of the doc linked by @fultonj) nor a linting issue.